how-we-work/mozdotorgdurable.md

```diff
@@ -1,74 +1,123 @@
 
 ## Board Details
 
-* What are your columns, what do the columns mean?
+### What are your columns, what do the columns mean?
+
+Generally our projects will have the following columns:
+
+- `Backlog` - Tasks we know have to get done but are not necessarily ready to go.
+- `Ready` - Prioritized tasks that are ready to start.
+- `In Progress` - Tasks that are currently underway.
+- `Done` - Shipped!
 
-Board columns:  Incoming | Research | Ready | In progress | Review | Done | Archived 
+### How/when do cards move across the board?
 
-* How/when to cards move across the board?
+Cards are moved by any member of the team when a change in status is identified. Cards should generally be moved by whomever takes the work or causes an update to status. This is not the responsibility of the Program Manger, team members show own the cards they take on including keeping the status up to date.
 
-Cards are moved by any member of the team when a change in status is indentified. Chiefly this falls to the Program Manager, then to the individuals owning a card at the given moment until such time the state changes and it needs to move to another people resource and/or to another column.
+## Labels
 
-* What does done mean?
+We use a lot of labels, below are some special ones:
 
-Done means a fully approved (Accountable/Approver) work item is complete according to the established Acceptance Criteria. Most commonly, but not exclusively, this means the results of the body of work are live on mozilla.org or blog.mozilla.org.
+- `Blocked` - Tasks that can not move forward and require some action to get moving. 
+- `Needs Review` - Tasks that need review (creative/pr etc) to move forward. 
+- `Fire` -  Urgent request that should take priority over other work. This could be work that has legal, financial or firefox download ramifications. Fire requests should always be vetted with the Team Lead to determine priority.
+- `Project` - Should only be used for cards on the Project Board.
+- _* Team_ -  Should only be used for cards on the Project Board.
 
 
-## Daily Standups
+### Daily Standups
 
 *How / when to stand ups occur, what happens during a meeting, be brief!*
 
 * Stand up & Status - Mondays at 9-9:30 am PT - Stand up followed by project status review 
 * Stand ups - Tuesdays & Thursdays at 9-9:15 am PT
-* Team time - Wednesdays at 9-10 am PT - recurring, non-required meeting where we attempt realize the benefits of colocation (as we are not colocated)
+* Team time - Wednesdays at 9:15-10 am PT - recurring, non-required meeting where we attempt realize the benefits of colocation (as we are not colocated)
 
 
-## Retrospectives
+### Retrospectives
 
 Team Retrospectives are held every three weeks. These are 'team only' meetings, defined as safe spaces in which candid commentary is allowed and expected with the intent learn from previous circumstances
 We focus on: 
```

Comment:
-Did we follow up on Action Items from last retrospective?
-What do we keep doing?
-What could use improvements?
-What do we stop doing?